tests/specs/framework-detection.e2e.js

```diff
@@ -0,0 +1,61 @@
+import { supportsFetchExtended } from '../../tools/browser-matcher/common-matchers.mjs'
+import { getMetricsFromResponse } from './helpers/assertion-helpers'
+
+const config = {
+  init: {
+    page_view_timing: {
+      enabled: false
+    }
+  }
+}
+
+describe('framework detection', () => {
+  withBrowsersMatching(supportsFetchExtended)('detects a page built with REACT and sends a supportability metric', async () => {
+    const rumPromise = browser.testHandle.expectRum()
+    await browser.url(await browser.testHandle.assetURL('frameworks/react/simple-app/index.html', config)).then(() => browser.waitForAgentLoad())
+
+    await rumPromise.then(async () => {
```

Comment:
Could write this using `Promise.all`

```suggestion
    const [rumResult] = await Promise.all([
      browser.testHandle.expectRum(),
      browser.url(await browser.testHandle.assetURL('frameworks/react/simple-app/index.html', config))
        .then(() => browser.waitForAgentLoad())
    ])
```